pyrobuf/parse_proto.py

```diff
@@ -252,25 +234,34 @@ def parse(self, cython_info=True, fname='', includes=None):
         return rep
 
     @classmethod
-    def parse_from_filename(cls, fname, includes):
+    def parse_from_filename(
```

Comment:
I don't love this style for function declarations, but I guess it's fine.

Reply:
I did the line breaks because it was over the 80 line limit I saw the file written with. Anyway, after renaming the variable it is now a one line definition again :)